store/tikv/client.go

```diff
@@ -69,7 +69,7 @@ func newRPCClient() *rpcClient {
 	}
 }
 
-func (c *rpcClient) callRpcFunc(addr string, f func(conn *Conn), label string) error {
+func (c *rpcClient) callRPCFunc(addr string, f func(conn *Conn), label string) error {
```

Comment:
I prefer using call or callFunc, no need using RPC in function name because the structure has already indicated its RPC.